addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
OK. Label now reads ```Set as default for all media```

For the sake of clarity. without including what DSP saves or not as I dont know. But non DSP wise.

Volume, volume amplification, audio or subtitle offsets are not saved for all files or for any near as I can tell...

Pixel ratio affects View mode immediately both of these are saved for all media, causes mayhem deinterlacing settings and method and scaling method also get saved for all media.

There are a few unknowns like post-processing, brightness/contrast, vdpau sharpness/ vdpau contrast and others which not so destructive and Im unsure if they saved for all of not.
Same with calibration I think that affects all...

Its a soup to say the least now with dsp range of settings into this thers even more unknowns like what happens if you save settings and dsp is then disabled

For instance DSP duplicated code for audio delay subtitle delay dialogs, already I know they dont behave same as existing ones but if you save them what happens then? no idea.

Very dangerous setting as far as Im concerned due to all unknowns and damage it can do and rather undocumented.